backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -289,6 +288,8 @@
                                         .call(session::flush)
                                         .onItem().transform(rowCount -> rowCount > 0);
                             case EMAIL_SUBSCRIPTION:
+                                LOGGER.warning("Endpoint updating properties of EmailSubscription. It should never happen.");
+                                // Todo: Should throw? EmailSubscriptions are not updatable once they are created
```

Comment:
Even if there's a check in `EndpointService` which should prevent this case, throwing would make sense, but the endpoint itself (not only its properties) can't be updated so I would throw at the beginning of this method.